internal/scorecard/alpha/scorecard.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alpha
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/operator-framework/operator-sdk/version"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+var (
+	Log = logrus.New()
+)
+
+type ScorecardFlags struct {
+	Config       string
+	Bundle       string
+	Selector     string
+	ListAll      bool
+	OutputFormat string
+	Kubeconfig   string
+}
+
+// RunTests executes the scorecard tests as configured
+func RunTests(flags ScorecardFlags) error {
+	scConfig, err := lookupConfig(flags)
+	if err != nil {
+		return err
+	}
+
+	selector, err := labels.Parse(flags.Selector)
+	if err != nil {
+		return err
+	}
+
+	tests := selectTests(selector, scConfig.Tests)
+
+	for i := 0; i < len(tests); i++ {
+		if err := runTest(tests[i]); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// lookupConfig will find and return the scorecard config, the config file
+// can be passed in via command line flag or from a bundle location or
+// bundle image
+func lookupConfig(flags ScorecardFlags) (ScorecardConfig, error) {
```

Comment:
Nit:
```suggestion
func LoadConfig(configFile string) (ScorecardConfig, error) {
```

For the case of loading from the bundle image, we will probably extract the bundle image somewhere on disk and still load the file from there. But since we may want to do other things with the bundle (e.g. make a ConfigMap from it), it probably makes sense to separate the bundle fetching and loading from the config file loading.

WDYT?

Reply:
updated the function name in the latest commit.